Makefile

```diff
@@ -245,13 +249,64 @@ releases/$(RELEASE).tar.gz: $(BINARIES)
 	@tar -czf releases/$(RELEASE).tar.gz -C releases/$(RELEASE) bin
 	@rm -rf releases/$(RELEASE)
 
-release: $(BINARIES) releases/$(RELEASE).tar.gz
+release: releases/$(RELEASE).tar.gz
 	@echo "$(WHALE) $@"
 	@cd releases && sha256sum $(RELEASE).tar.gz >$(RELEASE).tar.gz.sha256sum
 
+ifeq ($(GOOS),windows)
+install-cri-deps: $(BINARIES)
+	mkdir -p $(CRIDIR)
+	DESTDIR=$(CRIDIR) script/setup/install-cni-windows
+	cp bin/* $(CRIDIR)
+else
+install-cri-deps: $(BINARIES)
+	@sudo rm -rf ${CRIDIR}
+	@sudo install -d ${CRIDIR}/usr/local/bin
+	@sudo install -D -m 755 bin/* ${CRIDIR}/usr/local/bin
+	@sudo install -d ${CRIDIR}/opt/containerd/cluster
+	@sudo cp -r contrib/gce ${CRIDIR}/opt/containerd/cluster/
+	@sudo install -d ${CRIDIR}/etc/systemd/system
+	@sudo install -m 644 containerd.service ${CRIDIR}/etc/systemd/system
+	echo "CONTAINERD_VERSION: '$(VERSION:v%=%)'" | sudo tee ${CRIDIR}/opt/containerd/cluster/version
+
+	DESTDIR=$(CRIDIR) USESUDO=true script/setup/install-runc
+	DESTDIR=$(CRIDIR) script/setup/install-cni
+	DESTDIR=$(CRIDIR) script/setup/install-critools
+
+	@sudo install -d $(CRIDIR)/bin
+	@sudo install $(BINARIES) $(CRIDIR)/bin
+endif
+
+ifeq ($(GOOS),windows)
+releases/$(CRIRELEASE).tar.gz: install-cri-deps
+	@echo "$(WHALE) $@"
+	@cd $(CRIDIR) && tar -czf ../../releases/$(CRIRELEASE).tar.gz *
+
+releases/$(CRICNIRELEASE).tar.gz: install-cri-deps
+	@echo "$(WHALE) $@"
+	@cd $(CRIDIR) && tar -czf ../../releases/$(CRICNIRELEASE).tar.gz *
+else
+releases/$(CRIRELEASE).tar.gz: install-cri-deps
+	@echo "$(WHALE) $@"
+	@tar -czf releases/$(CRIRELEASE).tar.gz -C $(CRIDIR) etc/crictl.yaml etc/systemd usr opt/containerd
+
+releases/$(CRICNIRELEASE).tar.gz: install-cri-deps
+	@echo "$(WHALE) $@"
+	@tar -czf releases/$(CRICNIRELEASE).tar.gz -C $(CRIDIR) etc usr opt
+endif
+
+cri-release: releases/$(CRIRELEASE).tar.gz
+	@echo "$(WHALE) $@"
+	@cd releases && sha256sum $(CRIRELEASE).tar.gz >$(CRIRELEASE).tar.gz.sha256sum
+
+cri-cni-release: releases/$(CRICNIRELEASE).tar.gz
+	@echo "$(WHALE) $@"
+	@cd releases && sha256sum $(CRICNIRELEASE).tar.gz >$(CRICNIRELEASE).tar.gz.sha256sum
+
 clean: ## clean up binaries
 	@echo "$(WHALE) $@"
 	@rm -f $(BINARIES)
+	@rm -rf _output
```

Comment:
`$(ROOTDIR)_output` ?

Reply:
Updated here and uses sudo when it exists to remove since it gets created with sudo.